dbux-code/src/webViews/pathwaysDecorations.js

```diff
@@ -35,7 +35,7 @@ export async function decorateVisitedTraces(pdp) {
 
   const staticTracesByFile = pdp.util.getVisitedStaticTraces();
   for (const [fpath, staticTraces] of staticTracesByFile.entries()) {
-    const decos = staticTraces?.map(staticTrace => {
+    const decos = Array.from(staticTraces)?.map(staticTrace => {
```

Comment:
`Array.from`...
* never returns a falsy value, so the `?` is unnecessary.
* only accepts iterables. So if `staticTraces` might be falsy, need to check explicitely beforehand.

Reply:
I added a sanity check and removed the ?, staticTraces should always be a Set instead of undefined.